lib/components/NewMultisigWallet.js

```diff
@@ -87,8 +99,7 @@ class NewMultisigWallet extends PureComponent {
     history.goForward();
   }
 
-  update(key, _value) {
-    const value = _value && _value.toString();
```

Comment:
I needed a boolean value for the html `disabled` which is why I got rid of this line